x-pack/plugins/security_solution/public/overview/components/entity_analytics/header/index.tsx

```diff
@@ -88,7 +88,7 @@ export const EntityAnalyticsHeader = () => {
     return [onClick, href];
   }, [dispatch, getSecuritySolutionLinkProps]);
 
-  const totalAnomalies = useMemo(() => sum(data.map(({ count }) => count)), [data]);
+  const totalAnomalies = useMemo(() => sum(data.map(({ count }) => count)) || '-', [data]);
```

Comment:
Do we want to show ` - ` if there are `0` total anomalies? 🤔 

btw, we could also do: `sumBy('count', data);`

Reply:
Good question. It isn't clear when we should display the dash because anomalies are not a single module. They are just a group of jobs. So I updated this logic only to show the dash if all jobs are uninstalled. Otherwise, it will display `0`. I think that it is more consistent this way.

Reply:
Oh now I see the latest changes, yeah that's exactly what I meant. Thanks! 💯